src/ray/common/ray_syncer/ray_syncer.cc

```diff
@@ -147,6 +152,12 @@ void RaySyncer::Disconnect(const std::string &node_id) {
   boost::asio::dispatch(io_context_.get_executor(), std::move(task)).get();
 }
 
+void RaySyncer::SetCompletedRpcCallbackForOnce(CompletedRpcCallback on_rpc_completion) {
```

Comment:
Can we pass this callback to ray syncer constructor instead of setting it later at a different time.

Reply:
Updated.